client/src/pages/CreateUserProfile.js

```diff
@@ -0,0 +1,219 @@
+    margin: var(--mt) 0 0 0;
+    p {
+        color: ${theme.colors.darkerGray};
+        font-weight: normal;
+    }
+    p, div {
+        margin-left: ${theme.typography.size.xsmall};      
+    } 
+`
+
+const CheckboxGroup = ({children, description, label}) => {
+    return (
+    <CheckboxContainer>
+        <Checkbox/>
+        <div>
+            {children ?? <Label label={label}/>} 
+            <UnderlineDescription>{description}</UnderlineDescription>
+        </div>
+    </CheckboxContainer>
+    )
+};
+
+const StyledMenu = styled(Menu)`
+    padding: ${theme.typography.size.xxsmall};
+    width: 100%;
+    .ant-dropdown-menu-item {
+        display: flex;
+        span {
+            font-size: ${theme.typography.size.medium};
+            margin-right: ${theme.typography.size.xxsmall};
+            :first-child {
+                color: ${theme.colors.darkerGray};
+                font-weight: 500;
+            }
+            :last-child {
+                margin: 0;
+            }
+        }
+        &.ant-dropdown-menu-item-active {
+            background-color: ${theme.colors.selago};
+        }
+    }
+`;
+
+const DropdownMenu = ({children}) => {
+    const menu = (
+        <StyledMenu>
+            <Menu.Item>
+               <span>10014</span>
+               <span>13th Ave,</span>
+               <span>New York,</span>
+               <span>NY,</span>
+               <span>USA</span>
+            </Menu.Item>
+            <Menu.Divider/>
+        </StyledMenu>
+    );
+    return <Dropdown overlay={menu}>
+        {children}
+    </Dropdown>
+}
+
+const InputGroup = styled.div`
+    --py: ${theme.typography.size.xxsmall};
+    --my: ${theme.typography.size.xxxlarge};
+    
+    margin: var(--my) 0 var(--my) 0;
+
+    h3 {
+        color: ${theme.colors.primary};
+        font-weight: bold;
+    }
+    input {
+        color: ${theme.colors.darkerGray};
+        font-size: ${theme.typography.size.xlarge};
+        width: 100%;
+        margin-bottom: var(--my);
+        padding: var(--py) 0 var(--py) 0;
+
+    }
+    label {
+        font-family: ${theme.typography.font.family.button};
+        font-size: ${theme.typography.size.large};
+        font-weight: 500;
+    }
+`
+
+const UnderlineLink = styled.p`
+    --color: ${theme.colors.darkGray};
+    color: var(--color) !important;
+    a {
+        color: var(--color);
+        text-decoration: underline;
+    }
+    font-size: ${theme.typography.size.small};
+`;
+
+const CreateProfile = () => {
+return <Container>
+    <Flex className="image-container" direction="column">
+        <img src={PersonalDataImage} alt="two people standing next to a list of personal items" />
+    </Flex>
+    <Flex className="form-container" direction="column">
+        <WhiteSpace size="xl"/>
+        <ProfileFormGroup>
+            <CustomH1 color={theme.colors.darkerGray} fontweight="bold">Create your Profile</CustomH1>
+            <InputGroup>
+                <TextInput label="E-mail" />
```

Comment:
- I'm not sure if it's a mistake by the design team but only the email field is a gray while the other inputs are royal blue.
- The underline moves surrounding elements when it's active/hovered/focused
- Need the icon that's next to the Address label

figma:
![Screen Shot 2020-05-07 at 2 49 49 PM](https://user-images.githubusercontent.com/42228386/81348118-5847e680-9072-11ea-8985-974e24bd1d19.png)

Reply:
Yeah, on the Desktop MVP in Figma, there appears to be a screen where the email field is grayed out, and another screen where it is not while the location drop-down menu is open. I am not sure if that is intentional. 

![image](https://user-images.githubusercontent.com/11316853/81350682-546a9300-9077-11ea-8fd5-50f1e33d1048.png)

Reply:
Actually on this screen, the email field is removed entirely and the email address is displayed in a welcome message as a subheading. I'm assuming this is the intended behavior after a user enters their email address.